pkg/jaeger/jaeger_event_test.go

```diff
@@ -0,0 +1,96 @@
+package jaeger
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
```

Comment:
Spacing here looks odd.

Reply:
`make format` is performed - so looks like it is the formatting rules?

Reply:
Strange, it should have kept the `github.com/jaegertracing` together with the `github.com/stretchr`. If you manually remove the empty lines between those two imports, does `make format` work correctly?